python/lsst/afw/image/_exposureSummaryStats.py

```diff
@@ -155,11 +155,9 @@ def _read(bytes):
         if len(droppedFields) > 0:
             droppedFieldString = ", ".join([str(f) for f in droppedFields])
             warnings.warn(
```

Comment:
Should this be a `log.warning` instead?

Reply:
This one does feel more like a `warnings.warning` to me, just because it's something about backwards compatibility.